src/core_landice/shared/mpas_li_constants.F

```diff
@@ -62,11 +62,11 @@ module li_constants
                                                            !< http://www.csgnetwork.com/h2ofreezecalc.html (25 Nov. 2014)
 
    real (kind=RKIND), parameter, public :: &
-        iceMeltingPointPressureDependence = 9.7456e-8_RKIND  ! Dependence of ice melting point on pressure (K Pa^-1)
+        iceMeltingPointPressureDependence = 0.0e-8_RKIND  ! Dependence of ice melting point on pressure (K Pa^-1)
```

Comment:
Ahh sorry, this is for the benchmark experiment B. I forgot to change it back!

Reply:
Thanks for removing the parameter changes that were required for exp B.  I don't see that experiment in COMPASS - I don't remember how you set it up and ran it.  If it's not too much additional work, it would be good to add the test case to the PR too.  I can help you do that if you want.

Reply:
Yes, we should do that. I have the code and working directories around. But I am not familiar with adding example in COMPASS. It would be great if I can get some helps from you.

Reply:
I've changed the value back to the original value